public/app/features/logs/components/PopoverMenu.tsx

```diff
@@ -84,8 +84,9 @@ export const PopoverMenu = ({
   );
 };
 
-function track(event: string, selectionLength: number, dataSourceType: string | undefined) {
-  reportInteraction(`grafana_explore_logs_${event}`, {
+function track(action: string, selectionLength: number, dataSourceType: string | undefined) {
+  reportInteraction(`grafana_explore_logs_popover_menu`, {
+    action,
     selection_length: selectionLength,
     ds_type: dataSourceType || 'unknown',
```

Comment:
Just fyi, we usually use `datasourceType` as a name for property.

Reply:
Automerged before I saw this comment. I'll sent another one with that change, thanks!